### PR TITLE
Add \lT command listing scalar types.

### PR DESCRIPTION
**Pull request by vpetrovykh**
Add `\lT [PATTERN]` command for listing scalar types, optionally
filtered by name using the specified regexp pattern.
Add `\lTI [PATTERN]` command for case-sensitive pattern matching.

Issue #179

### CODE REVIEW

File: mypy.ini [hunk 1]
```diff
@@ -140,3 +140,11 @@ ignore_errors = False
 [mypy-edb.tools.profiling]
 follow_imports = True
 ignore_errors = False
+
+[mypy-edb.repl.table]
```
Comment (elprans) [inline mypy.ini]: Let's use strict mode for new code

Comment (vpetrovykh) [reply mypy.ini]: Updated.

File: edb/repl/__init__.py [hunk 2]
```diff
@@ -362,6 +363,80 @@ def _command_simplelist(self, pattern: str, query: str,
...
 
+    @_command('lT', R'\lT [PATTERN]', 'list scalar types')
+    def command_list_scalar_types(self, args: str, *,
+                                  case_sensitive: bool = False) -> None:
+        self._list_scalar_types(args.strip(), case_sensitive=case_sensitive)
+
+    @_command('lTI', R'\lTI [PATTERN]',
+              'list scalar types (case sensitive pattern match)')
+    def command_list_scalar_types_i(self, args: str) -> None:
+        self.command_list_scalar_types(args, case_sensitive=True)
```
Comment (1st1) [inline edb/repl/__init__.py]:
```suggestion
        self._list_scalar_types(args.strip(), case_sensitive=True)
```

File: edb/repl/__init__.py [hunk 3]
```diff
@@ -362,6 +363,80 @@ def _command_simplelist(self, pattern: str, query: str,
                 print(f'No {item_name} found matching '
                       f'{eql_quote.quote_literal(pattern)}')
 
+    @_command('lT', R'\lT [PATTERN]', 'list scalar types')
+    def command_list_scalar_types(self, args: str, *,
+                                  case_sensitive: bool = False) -> None:
```
Comment (1st1) [inline edb/repl/__init__.py]:
```suggestion
```

File: edb/repl/__init__.py [hunk 4]
```diff
@@ -362,6 +363,80 @@ def _command_simplelist(self, pattern: str, query: str,
...
+        self._list_scalar_types(args.strip(), case_sensitive=case_sensitive)
```
Comment (1st1) [inline edb/repl/__init__.py]:
```suggestion
        self._list_scalar_types(args.strip(), case_sensitive=False)
```

File: edb/repl/table.py [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+
+class ColumnSpec(NamedTuple):
+    field: str
+    title: str
+    width: int
+    align: str
+
+
+def render_table(
+    title: str,
```
Comment (1st1) [inline edb/repl/table.py]: I'd make them all keyword-only

File: edb/repl/table.py [hunk 6]
```diff
@@ -0,0 +1,170 @@
...
+    data: Iterable[edgedb.Object],
+    max_width: int = 0,
+    *,
+    file: typing.TextIO = sys.stdout,
+) -> None:
+    if max_width == 0:
+        return
+
+    # find the amount of space available for text
+    avail_width = max(0, max_width - (3 * len(columns) - 1))
```
Comment (1st1) [inline edb/repl/table.py]: Pls add a comment to explain this formula

File: edb/repl/table.py [hunk 7]
```diff
@@ -0,0 +1,170 @@
...
+import typing  # needed for linting TextIO
+
+import edgedb
+
+
+class ColumnSpec(NamedTuple):
+    field: str
+    title: str
+    width: int
+    align: str
```
Comment (1st1) [inline edb/repl/table.py]: Use https://mypy.readthedocs.io/en/latest/literal_types.html#literal-types -- a literal type with `left`, `right`, `center` values

Comment (vpetrovykh) [reply edb/repl/table.py]: Done

File: edb/repl/table.py (hunk 6 above)
Comment (vpetrovykh) [reply edb/repl/table.py]: Comment added.

File: edb/repl/table.py (hunk 5 above)
Comment (vpetrovykh) [reply edb/repl/table.py]: Sure.

**Pull request merged**